website/docs/components/form/primitives/partials/code/component-api.md

```diff
@@ -34,6 +34,29 @@
   </C.Property>
 </Doc::ComponentApi>
 
+### Form::CharacterCount
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="controlId" @type="string">
+    The ID of the form control associated with the helper text. This is used to populate the `id` HTML attribute (with a `character-count-` prefix) of the element. This CharacterCount ID can then be referenced in the `aria-describedby` attribute of the form control.
+  </C.Property>
+  <C.Property @name="maxLength" @type="number">
+    The maximum number of characters allowed in the associated form element, used to determine the number of remaining characters. This does not restrict users from adding characters over the limit.
```

Comment:
Should we mention how to restrict user if needed?

Reply:
I think it would be helpful to mention.

Reply:
I'm personally not keen on recommending people to use the `maxlength` attribute on input fields, but sure, I can extend the property description to also explain the attribute.

Reply:
I see. But I would imagine this would be a not uncommon request, so in any case we should provide guidance on how to achieve the desired result (prevent a user from entering more characters than the `maxLength`). Unless is a specific design/UX decision (@jorytindall @andgen404 ?) in which case we should explain its rationale so consumers are aware of this.